src/uct/cuda/gdr_copy/gdr_copy_md.c

```diff
@@ -251,22 +251,40 @@ uct_gdr_copy_mem_dereg(uct_md_h uct_md,
     return status;
 }
 
+static int uct_gdr_is_gdr_get_info_v1(int major, int minor)
+{
+    return (major < 2) || ((major == 2) && (minor <= 3));
+}
+
 static ucs_status_t
 uct_gdr_copy_query_md_resources(uct_component_t *component,
                                 uct_md_resource_desc_t **resources_p,
                                 unsigned *num_resources_p)
 {
+    int major, minor;
     gdr_t ctx;
 
+    gdr_runtime_get_version(&major, &minor);
```

Comment:
maybe question of taste, but IMO it will be more reusable from other functions if we encapsulate the whole block into a separate (inline?) function `uct_gdr_is_compatible` or something. Especially if in the future we want to support multiple versions

Reply:
using separate function. not sure what reusability you mean, since we have one use-case. in the future, when we know what is needed we will anyway need to change all that.